### DISCUSSION TITLE
My Organizatin has been flagged.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#28116] [Other Feature Feedback, Questions, & Ideas]
The NFS-projects organization has been flagged.
Because of that, your organization is hidden from the public. If you believe this is a mistake, [contact support](https://github.com/contact) to have your organization’s status reviewed.

maybe because I used the wrong token

**Comment by airtower-luna** [comment]
Us other users won't be able to say, you'll need to [create a support ticket](https://support.github.com/request) like the message says so GitHub staff can check. Good luck!
